config/auto_battle_operation/般岳-招架支援-铁壁.sample.yml

```
@@ -0,0 +1,13 @@
+# 般岳-招架支援-铁壁
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)